src/PresentationalComponents/Portfolio/PorfolioCard.js

```diff
@@ -20,7 +20,7 @@ const createToolbarActions = (portfolioName, portfolioId) => [
       <EditAltIcon fill={ ICON_FILL } />
     </Button>
   </Link>,
-  <Link key="remove-portfolio-action" to={ `/portfolios/remove/${portfolioId}` }>
+  <Link key="remove-portfolio-action" to={ `/portfolio/${portfolioId}/remove-portfolio` }>
```

Comment:
This URL will redirect you to portfolio detail. Is this intentional? If you want to stay on the list, you will have to add new route (with the same modal component) and use different path.

see: https://github.com/ManageIQ/service_portal-ui/blob/27292383e6daac00dc9c5eeee483de8cf707a538/src/PresentationalComponents/Portfolio/PorfolioCard.js#L14

and: https://github.com/ManageIQ/service_portal-ui/blob/27292383e6daac00dc9c5eeee483de8cf707a538/src/SmartComponents/Portfolio/Portfolios.js#L78